### ISSUE TITLE
Account Sync (hard refresh)

### ISSUE THREAD

**Issue by cakesoft-swati** [bithyve/hexa#2820]
-Ideally this should be on the Account Screen.
<img width="375" alt="D6CB795B-9B1D-4B42-A8D0-92B16D7AED78" src="https://user-images.githubusercontent.com/62699947/109505124-497b6600-7ac2-11eb-81c4-d68cffcc870a.png">

-After clicking the Continue button of "Rescan Your Account" of Manual Rescan, "Scan Your Account" pops up after 6/7 seconds.Until then, the user does not understand that some activity is running in the background or the screen is stuck.

-This section is ok when the scanning is in progress. But once it is done, The message should change to “Account Synced” and it should be in bold blue colour.
![File (33)](https://user-images.githubusercontent.com/62699947/109804893-cc2e2d80-7c48-11eb-9c30-21199d445053.jpg)

-Account Sync in progress
![File (34)](https://user-images.githubusercontent.com/62699947/109804957-dfd99400-7c48-11eb-8293-f43b91135483.jpg)

“Account synced”
![File (35)](https://user-images.githubusercontent.com/62699947/109805006-ef58dd00-7c48-11eb-938d-92ba87609dd1.jpg)

**Comment by AliMeer** [comment]
Done except the firs part, created new issue for first part: https://github.com/bithyve/hexa/issues/3184

**Comment by cakesoft-devika** [comment]
-Verified this issue on Dev app v1.7.0 (286)
